backend/benefit/applications/api/v1/serializers.py

```diff
@@ -1124,10 +1125,19 @@ def _update_de_minimis_aid(self, application, de_minimis_data):
             ] = idx  # use the ordering defined in the JSON sent by the client
         serializer.save()
 
+    def _get_request_user_from_context(self):
+        request = self.context.get("request")
+        if request:
+            return request.user
+        return None
+
     def logged_in_user_is_admin(self):
-        # TODO: user management
+        user = self._get_request_user_from_context()
+        if user:
+            return user.is_handler()
         return False
 
     def get_logged_in_user_company(self):
```

Comment:
Is / will this be used somewhere?

Reply:
Yes, they are already used here: https://github.com/City-of-Helsinki/yjdh/pull/261/files#diff-91a79cb2d83133bd53a5d3272d60fe0e61c2bdc63af91389490d7cf661cbb849R1085. But they were only place-holder previously.